glean_parser/templates/kotlin.jinja2

```diff
@@ -7,34 +7,45 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+{% macro metric_declaration(metric, suffix='', access='') %}
```

Comment:
nit: do we need a blank line above this?

Reply:
I wish we could.  If we have one, you end up with two blank lines in a row in the output and `ktlint` complains :(

Reply:
Gah :S Nevermind then!